src/components/elements/annotationSidePanel/components/annotationWizard/steps/ExistingInstance.tsx

```diff
@@ -111,7 +111,7 @@ const ExistingInstance = (props: Props) => {
                                 </Row>
                             </div>
                                 : <div>
-                                    <p className="fs-4"><span className="fw-lightBold">Value:</span> {instanceValue}</p>
+                                    <p className="fs-4"><span className="fw-lightBold">Value:</span> {String(instanceValue)}</p>
```

Comment:
do you need catch a type exception if `instanceValue` can't be mapped to a string?

Reply:
Instance value should always be convertable to a string, begin a simple data type, the complicated data types are filtered in the step before